src/main/java/uk/gov/hmcts/reform/ccd/service/remote/DisposeElasticsearchRemoteOperation.java

```diff
@@ -58,38 +54,28 @@ public void delete(final CaseData caseData) {
     }
 
     private void deleteByQueryRequest(final DeleteByQueryRequest request) throws IOException {
+        try {
+            final DeleteByQueryResponse response = elasticsearchClient.deleteByQuery(request);
 
-        final RequestOptions requestOptions = buildRequestOptions();
-        final BulkByScrollResponse bulkResponse = elasticsearchClient.deleteByQuery(request, requestOptions);
-
-        final List<ScrollableHitSource.SearchFailure> searchFailures = bulkResponse.getSearchFailures();
-        final List<BulkItemResponse.Failure> bulkFailures = bulkResponse.getBulkFailures();
-
-        if (!isEmpty(searchFailures)) {
-            throwError(SEARCH_FAILURES, searchFailures);
-        }
-        if (!isEmpty(bulkFailures)) {
-            throwError(ELASTICSEARCH_FAILURES, bulkFailures);
+            if (!isEmpty(response.failures())) {
+                throwError(SEARCH_FAILURES, response.failures());
+            }
+        } catch (ElasticsearchException e) {
```

Comment:
This is very narrow exception handling and we definitely might miss an exception. I'd recommend following the pattern of the existing "Dispose" classes.

Reply:
try and catch has been removed